src/components/panel/panel.js

```diff
@@ -306,6 +309,8 @@ angular
  * Adds a class to the panel. DO NOT use this to hide/show the panel.
  *
  * @param {string} newClass Class to be added.
+ * @param {boolean} toElement Whether or not to add the class to the panel
```

Comment:
I am concerned that the documentation standards are widely diverging for our ngM1 components.